### PR TITLE
www/unit: add new port

### PR DESCRIPTION
**Pull request by osokin**
#### Description

Add www/unit, version 1.25.0

###### Type(s)
- [X] new submission

###### Tested on

macOS 11.6 20G165 x86_64
Xcode 12.5.4

###### Verification 
Have you

- [X] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [X] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [X] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [X] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [X] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?  No tests
- [X] tried a full install with `sudo port -vst install`?
- [X] tested basic functionality of all binary files?

### CODE REVIEW

File: www/unit/Portfile [hunk 1]
```diff
@@ -0,0 +1,220 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+
+name            unit
+version         1.25.0
+categories      www
+platforms       darwin
+license         Apache-2
+maintainers     osokin
```
[the remark this answers is not in the document]
Comment (osokin) [reply www/unit/Portfile]: Fixed, thanks!

**Comment by reneeotten** [comment]
@osokin if you're still interested in getting this PR merged can you please check the CI failures related to the ruby subports? Secondly, I think we prefer not to add a large chunk of out-commented lines in the Portfile (i.e., the PHP stuff) - please either enable it of remove for now (of course you can keep it somewhere in your local repository if you want to continue working on that at a later time.

**Comment by osokin** [comment]
@reneeotten Sure, I'm still interested.  Need to update the port to the recent version of NGINX Unit 1.26.1 and re-submit it once again.   Also, I need some guidance from your side how to detect what's wrong with a remote builds because I can't reproduce that locally on my laptop.

**Comment by reneeotten** [comment]
> @reneeotten Sure, I'm still interested. Need to update the port to the recent version of NGINX Unit 1.26.1 and re-submit it once again. Also, I need some guidance from your side how to detect what's wrong with a remote builds because I can't reproduce that locally on my laptop.

okay, once you've update the portfile to the latest versions and push to this PR I will take a look

**Comment by reneeotten** [comment]
feel free to reopen this PR once you've had the time to finish this up!

**Pull request closed without merging**